README.md

```
@@ -12,6 +21,44 @@ The `CodeRev` is a web-based application designed to help programmers improve th
 
 ```
+2. Add this packages after directing to `trio-qbit` directory
+
+```bash
+dotnet add package Microsoft.EntityFrameworkCore
+dotnet add package Npgsql.EntityFrameworkCore.PostgreSQL # because of PostGreSQL
+dotnet add package Microsoft.EntityFrameworkCore.Tools
+dotnet add package Swashbuckle.AspNetCore # for Swagger support 
+dotnet add package Newtonsoft.Json # for Serialize string[] to JSON and Deserialize JSON to string[]
+```
+3. Add a `<name>.env` file into `/backend/db` directory with below contents
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)